vscode-dotty/src/worksheet.ts

```diff
@@ -23,11 +23,26 @@ export const worksheetRunKey = "dotty.worksheet.run"
  */
 export const worksheetCancelKey = "dotty.worksheet.cancel"
 
+/**
+ * A wrapper around the information that VSCode needs to display text decorations.
+ *
+ * @param decorationType    The styling options of this decoration
+ * @param decorationOptions The options of this decoraiton.
+ */
+class Decoration {
+  constructor(readonly decorationType: vscode.TextEditorDecorationType,
+              readonly decorationOptions: vscode.DecorationOptions) {
+  }
+}
+
 /** A worksheet managed by vscode */
 class Worksheet implements Disposable {
 
+  /** The version of this document the last time it was run */
+  private runVersion: number = 0
```

Comment:
Maybe use -1 since 0 could be a valid version ?

Reply:
I changed it